m/mongodb/Dockerfiles/rhel_8.3/Dockerfile

```diff
@@ -0,0 +1,135 @@
+FROM ppc64le/ubuntu:20.04
```

Comment:
the base image here is ubuntu:20.04 as opposed to RHEL or UBI, which is fine for now but we might have to think of creating an image with UBI base in future. For now, please correct the folder structure. This file should reside in an ubuntu_20.04 folder under Dockerfiles.

Reply:
Understood. Resolved.